ssm/crypto.py

```diff
@@ -180,6 +180,9 @@ def verify(signed_text, capath, check_crl):
         log.debug(error)
     else:
         log.warn(error)
+        raise CryptoException(
+            "Failed to verify the signed message, see the log for details."
```

Comment:
Do we need to say to check the log? Where would this message appear? We're already dumping out the error message in the previous line.

Reply:
I think the "see the log" message would only appear in the logs, so we don't need to say it. Removed.